test/unit/search/fm_index/bi_fm_index_aa27_test.cpp

```diff
@@ -8,6 +8,8 @@
 #include "fm_index_collection_test_template.hpp"
 #include "fm_index_test_template.hpp"
 
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
```

Comment:
Aren't we putting those above the relative includes? :nail_care: 
I don't care if not, just had the feeling